src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -667,13 +667,32 @@
         [:null name]
         [field name]))))
 
+(defn merge-function-options
+  "Optionally merge call and grouping into an existing query map.
+   Alias function calls with escape-quoted function name for hsqldb compatability.
+   For instance, (merge-function-options {} ['count' :*] ['status'])"
+  [selection call grouping]
+  (cond-> selection
+    call (update-in [:select] conj [(apply hcore/call call) (format "\"%s\"" (first call))])
```

Comment:
Also I'm a little confused about `(format "\"%s\"" (first call))`, why can't you just do `(keyword (first call))`